EMBRACE_Client/Student.h

```diff
@@ -11,7 +11,7 @@
 @interface Student : NSObject {
     NSString* firstName;
```

Comment:
Need to update these variable names as well, since we're not collecting this information?